streaming_form_data/targets.py

```diff
@@ -107,6 +108,42 @@ def on_finish(self):
             self._fd.close()
 
 
+class DirectoryTarget(BaseTarget):
+    """DirectoryTarget writes (streams) the different input to an on-disk directory."""
```

Comment:
```suggestion
    """DirectoryTarget writes (streams) the different inputs to an on-disk directory."""
```

Reply:
Hello, I added the parser test as asked and made the suggested change. In addition, the code should be compliant with lint now :)

Reply:
Nice!